packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -167,6 +167,12 @@ const Mercury = WebexPlugin.extend({
 
     Promise.all([this._prepareUrl(socketUrl), this.webex.credentials.getUserToken()])
       .then(([webSocketUrl, token]) => {
+        if (!this.backoffCall) {
+          this.logger.info('mercury: prevent socket open when backOffCall no longer defined');
+
+          return Promise.reject(new Error('mercury: prevent socket open when backOffCall no longer defined due to abort?'));
```

Comment:
Handling the scenario where `_attemptConnection` was called, the `Promise.all(..)` started, a `disconnect()` runs, then the `Promise.all(..)` resolved. Without this check the code would still open the socket after a disconnect.